frontend/src/components/shared/answersetView/StoredAnswersetViewer.jsx

```diff
@@ -35,8 +35,12 @@ export default function StoredAnswersetView({ question_id, answer_id }) {
       return;
     }
 
+    const questionResponseJSON = JSON.parse(questionResponse);
+    const answerResponseJSON = JSON.parse(answerResponse);
```

Comment:
I'm not sure I understand here. Why were these parses added, and why weren't we doing this before?

Reply:
Ok cool. This sounds good.